src/graphql/node/crud.rs

```diff
@@ -313,12 +270,8 @@ pub fn get_node_settings(db: &Store) -> Result<Vec<Setting>> {
         let piglet: Option<ServerAddress> = if node_settings.piglet {
             Some(ServerAddress {
                 web: None,
-                rpc: Some(SocketAddr::new(
-                    node_settings
-                        .piglet_review_ip
-                        .unwrap_or_else(|| IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0))),
-                    node_settings.piglet_review_port.unwrap_or_default(),
-                )),
+                // Set to the default value since the review address fields has been removed.
+                rpc: Some(SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), 0)),
```

Comment:
Let's make this change explicit in changelog.md, that `get_node_settings` will be removed in the future, and it will return dummy value of 0.0.0.0:0. It is because it would be a very surprising change without any notice. It would be the same for other fields with the same comment.

Reply:
done